build.gradle

```diff
@@ -57,6 +57,11 @@ allprojects {
 
 	repositories {
 		mavenCentral()
+
+		maven {
+			url = "https://maven.parchmentmc.org"
```

Comment:
why did you place parchment repo here and not in the settings.gradle repositories block ?